third-party/CMakeLists.txt

```diff
@@ -173,6 +176,10 @@ endif()
 include(GetBinaryDeps)
 
 if(WIN32)
+  # curl.exe, ca-bundle.crt, tidy.exe, swiss.exe
+  GetBinaryDep(TARGET wintools
+    INSTALL_COMMAND ${CMAKE_COMMAND} -E copy_directory . ${DEPS_INSTALL_DIR}/bin)
+
```

Comment:
You can have multiple commands in the same call to GetBinaryDep e.g. so the copies can be done here

```cmake
GetBinaryDep(TARGET winpty
+  INSTALL_COMMAND ${CMAKE_COMMAND} -E make_directory ${DEPS_INSTALL_DIR}/bin
+    COMMAND ${CMAKE_COMMAND} -DFROM_GLOB=${DEPS_BUILD_DIR}/src/winpty/${ARCH}/bin/* -DTO=${DEPS_INSTALL_DIR}/bin/ -P ${CMAKE_CURRENT_SOURCE_DIR}/cmake/CopyFilesGlob.cmake
```

Another advantage of doing it here is that the binaries will be in the $PATH for other tasks like the CI tests.